lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -118,15 +180,14 @@ class window.Turbolinks
     triggerEvent 'page:receive'
     options.updatePushState ?= true
 
-    if doc = processResponse(xhr, options.partialReplace)
-      reflectNewUrl url if options.updatePushState
-      nodes = changePage(extractTitleAndBody(doc)..., options)
-      reflectRedirectedUrl(xhr) if options.updatePushState
+    doc = processResponse(xhr, options.partialReplace)
```

Comment:
I think it's optimizing for a pretty unlikely scenario to talk about the bundle being renamed (presumably, the helper we add will depend on the name of the asset to generate the data attribute), but I don't feel too strongly either way.